hacks/server/test_tunnel.sh

```
@@ -0,0 +1,287 @@
+# Check and set kubernetes context
+log_info "Checking kubernetes context..."
+CURRENT_CONTEXT=$(kubectl config current-context)
+if [ "$CURRENT_CONTEXT" != "$EXPECTED_CONTEXT" ]; then
+    log_error "Current context is $CURRENT_CONTEXT, expected $EXPECTED_CONTEXT"
+    log_error "Please switch to the correct context using: kubectl config use-context $EXPECTED_CONTEXT"
+    exit 1
+fi
+log_success "Using correct kubernetes context: $CURRENT_CONTEXT"
+
+log_info "Building Docker image..."
+cd $(git rev-parse --show-toplevel)  # Go to project root
```

Shellcheck (0.9.0-1)

[warning] Quote this to prevent word splitting.

(SC2046)